GliaWidgets/Public/Glia/Glia+Logging.swift

```diff
@@ -0,0 +1,88 @@
+extension Glia {
+    enum LoggerPhase {
+        case notConfigured(CoreSdkClient.Logger)
+        case configured(CoreSdkClient.Logger)
+    }
+
+    /// Logger that is used while logger from CoreSDK is not available,
+    /// when for example CoreSDK is not yet configured. This is simply
+    /// a wrapper around `Swift.print`.
+    /// This logger could be further extended to collect logs in memory
+    /// for example, and once logger configuration is to happen,
+    /// collected logs could be transferred to configured logger, but
+    /// this is out of scope for current implementation.
+    struct NotConfiguredLogger: CoreSdkClient.Logging {
+        var environment: Environment
+        var localLogger: (CoreSdkClient.Logging)?
+        var remoteLogger: (CoreSdkClient.Logging)?
+        var oneTime: CoreSdkClient.Logging { self }
+
+        func error(_ object: @autoclosure () -> Any, file: String, function: String, line: Int) {
+            environment.print(object())
```

Comment:
do we need to pass an `object` closure instead of an execution function and pass it as a value?
in sense of optiomization I think it is better to pass closure because in this case, Core SDK will be able to verify whether should it be logged or not. otherwise might be case when complex `object` with a bunch of formation will be executed here, but filter out on Core Sdk level.

Reply:
For this particular case it is not needed because this happens in `NotConfiguredLogger` which is just placeholder and wrapper around `print` function.